[BUGFIX] Rendre l'identifiant plus visible dans la double mire d'authentification SCO (PIX-18455)

EmmanuelleBonnemay opened the pull request:
## 🔆 Problème

Lorsqu'un élève qui souhaite passer une campagne est redirigé vers l'inscription (double mire sco), il renseigne dans la partie gauche ses nom, prénom, date de naissance, puis dans un second temps choisit de s'inscrire avec un identifiant ou un mot de passe.
L'identifiant, qui se trouve dans un champ prérempli, n'est pas assez visible (trop peu de contraste avec le fond).

<img src="https://github.com/user-attachments/assets/35cffb9b-1252-4a97-b1c4-4cca333eea73" width=411/>


## ⛱️ Proposition

Utiliser un PixInput `disabled` et en utilisant une couleur plus sombre pour faire ressortir la valeur du champ.

![Screenshot-PixApp-Username_more-prominent](https://github.com/user-attachments/assets/0dd6ee8e-f996-49b5-a3d8-b03cc214a1ec)

## 🌊 Remarques

RAS

## 🏄 Pour tester

- Reproduire les deux premières étapes du scénario de test de https://github.com/1024pix/pix/pull/12408 sur https://app-pr12707.review.pix.fr/campagnes/SCOBADGE1
- Constater que l'identifiant `harry.potter1212` est maintenant bien visible

EmmanuelleBonnemay committed:
fix(mon-pix): make prefilled username more prominent

Co-authored-by: LEGO Technix <109212476+lego-technix@users.noreply.github.com>

pix-bot-github commented:
Une fois les applications déployées, elles seront accessibles via les liens suivants :
  * [App (.fr)](https://app-pr12707.review.pix.fr)
  * [App (.org)](https://app-pr12707.review.pix.org)
  * [Orga (.fr)](https://orga-pr12707.review.pix.fr)
  * [Orga (.org)](https://orga-pr12707.review.pix.org)
  * [Certif (.fr)](https://certif-pr12707.review.pix.fr)
  * [Certif (.org)](https://certif-pr12707.review.pix.org)
  * [Junior](https://junior-pr12707.review.pix.fr)
  * [Admin](https://admin-pr12707.review.pix.fr)
  * [API](https://api-pr12707.review.pix.fr/api/)
  * [API MaDDo](https://pix-api-maddo-review-pr12707.osc-fr1.scalingo.io/api/)
  * [Audit Logger](https://pix-audit-logger-review-pr12707.osc-fr1.scalingo.io/api/)

Les variables d'environnement seront accessibles via les liens suivants :
  * [scalingo front](https://dashboard.scalingo.com/apps/osc-fr1/pix-front-review-pr12707/environment)
  * [scalingo api](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-review-pr12707/environment)
  * [scalingo api-maddo](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-maddo-review-pr12707/environment)
  * [scalingo audit-logger](https://dashboard.scalingo.com/apps/osc-fr1/pix-audit-logger-review-pr12707/environment)

theotime2005 reviewed (approved):
Je pense que le titre du commit devrait plutôt être fix si le nom de la PR est BUGFIX

bpetetot commented:
@EmmanuelleBonnemay Une solution plus "propre" et plus pérenne serait d'utiliser un composant `PixInput` avec l'état `disabled` au lieu d'en reproduire un avec des styles. Comme ça on sera sûr qu'il a toujours le bon style du design system.

Dans le fichier `mon-pix/app/components/routes/register-form.gjs`:

https://github.com/1024pix/pix/blob/24a01e8054af76b67148afb05906bd47739eccc6/mon-pix/app/components/routes/register-form.gjs#L169-L175

```js
<PixInput
  @id="username"
  @value={{this.username}}
  @requiredLabel={{true}}
  @disabled={{true}}
  type="text"
>
  <:label>{{t "pages.login-or-register.register-form.fields.username.label"}}</:label>
</PixInput>
```

Et on pourra supprimer le css qui ne sera plus utilisé.

À tester ;)

-------

**Remarque 2 :**

J'ai vu qu'il y avait une erreur dans le fichier `mon-pix/app/components/routes/register-form.gjs`: au niveau du `PixInput` de l'email on a `autocomplete="firstname"` au lieu de `autocomplete="email"`

https://github.com/1024pix/pix/blob/24a01e8054af76b67148afb05906bd47739eccc6/mon-pix/app/components/routes/register-form.gjs#L187

C'est l'occasion de corriger ce petit bug.

lego-technix committed:
fix(mon-pix): fix autocomplete attributes wrt context and spec

cf.
https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes/autocomplete#given-name
https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes/autocomplete#family-name

bpetetot reviewed (approved):
✅ Revu et testé

![image](https://github.com/user-attachments/assets/cbab9aff-9bc5-4796-b324-c46aa824e769)

pix-service commented:
:tada: This PR is included in version 5.154.0 :tada:

The release is available on [GitHub release](https://github.com/1024pix/pix/releases/tag/v5.154.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
